### ISSUE TITLE
`walkdir` not defined

### ISSUE THREAD

**Issue by JaredCrean2** [MichaelHatherly/Documenter.jl#61]
I got the following error:

```
Documenter: setting up build directory.
ERROR: LoadError: UndefVarError: walkdir not defined
 in exec at /home/jared/.julia/v0.4/Documenter/src/modules/Builder.jl:229
 in process at /home/jared/.julia/v0.4/Documenter/src/modules/Builder.jl:195
 in process at /home/jared/.julia/v0.4/Documenter/src/modules/Builder.jl:191
 in anonymous at /home/jared/.julia/v0.4/Documenter/src/Documenter.jl:120
 in cd at ./file.jl:22
 in makedocs at /home/jared/.julia/v0.4/Documenter/src/Documenter.jl:119
 in include at ./boot.jl:261
 in include_from_node1 at ./loading.jl:304
 in process_options at ./client.jl:280
 in _start at ./client.jl:378
```

when running `makedocs()`, using Julia version
```
Julia Version 0.4.4-pre+35
Commit 1a9429e* (2016-02-12 13:21 UTC)
Platform Info:
  System: Linux (x86_64-linux-gnu)
  CPU: Intel(R) Core(TM) i7 CPU       Q 820  @ 1.73GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Nehalem)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.3
```

**Comment by MichaelHatherly** [comment]
What version of `Compat` is installed? `walkdir` got backported and should be available in versions `0.7.17` and up of `Compat`. Packages that have been `Pkg.clone`d don't deal with dependencies automatically I think.

**Comment by JaredCrean2** [comment]
I had `0.7.12`, updating it fixed things.  Thanks for the quick reply.
